index.js

```diff
@@ -30,11 +33,90 @@ module.exports = function(filename, root, visited) {
 
   var results = traverse(filename, root, visited);
   results = removeDups(results);
+  var tree = {
```

Comment:
Actually, your way is better for lookups since the nodes object is flat. The way above is what I had in mind for https://github.com/mrjoelkemp/sublime-dependents/issues/55 where the dependency tree is displayed as a JSON file.